panoptes_aggregation/scripts/extract_panoptes_csv.py

```diff
@@ -100,6 +100,8 @@ def extract_csv(
         version_range = config_yaml['workflow_version']
         for key, value in version_range.items():
             version_range[key] = packaging.version.parse(value)
+            if key == 'max' and version_range[key].minor == 0:
+                version_range[key] = packaging.version.parse(str(version_range[key].major + 1))
```

Comment:
Hmm, I think this might have unintended consequences with line 135 when `<=` is used in the comparison.  Would it be better to bump the major like this (or the minor if given) and change 135 to a `<`?

Reply:
I think this should work as intended, it's actually just duplicating the same logic that is used in the else block (when only one version number is specified). But maybe it should be documented better?

Reply:
I just double-checked, a workflow can't have a minor version of `0` on the Zooniverse so the edge case I was thinking of can't happen.

Originally I was thinking of the case where you wanted all workflows with major `10` but had another workflow with version `11.0`, the code as is would also catch this unwanted workflow.  But within the Zooniverse, the lowest allowed minor workflow version is `1` so it is not an issue.